README.md

```diff
@@ -8,19 +8,19 @@
 [![Slack](https://slack.annict.com/badge.svg)](https://slack.annict.com)
 
 
-**This branch is still under development for next version (V2).**
+**This branch is still in development for the next version (V2).**
 
 ### Contributing
 
 #### Requirements
 
-To run Annict on local machine, you need to install some software below:
+To run Annict on a local machine, you need to have the following dependencies installed:
```

Comment:
True, although I didn't want to change the original text too much. I was struggling with what to do with this line for a while though. 😅